src/lib/jsonParseV2/utilsParse.h

```diff
@@ -38,6 +38,22 @@ struct Opt {
   bool ok() { return error.empty(); }
 };
 
+
+
+/*
+ * getBoolMust -  get a mandatory bool from the rapidjson node
+ */
+Opt<bool> getBoolMust(const rapidjson::Value& parent, const char* field,const std::string& description = "");
```

Comment:
`getBoolMust` and `getBoolOpt` were initially added when the approach was to use a true/false flag for metadata field. Now that the field has evolved to an array, they are no longer needed.

However, I think we can leave them as utility functions, to be used to process future boolean field at parsing stage.

Opinions?